apps/api/src/schema.gql

```diff
@@ -141,6 +142,7 @@ type ProposalMetadataItem {
   body: Text
   discussion: Text
   execution: Text
+  labels: Text
```

Comment:
Any particular reason this is stored as Text instead of just simple array of String?

Reply:
There is no reason, just to keep it the same as `execution` where array is a string 😄 but yea execution may have some other properties. i will try to change it to array of string